etc/config/wazuh.conf

```diff
@@ -1,8 +1,6 @@
 [agent]
-server_mgmt_api_port = "55000"
-agent_comms_api_port = "27000"
-manager_ip = "localhost"
-https_enabled = "yes"
+server_url = https://localhost:27000
+registration_url = https://localhost:55000
```

Comment:
the value of `server_url `and `registration_url `must be enclosed in double quotation marks:

```
server_url = "https://localhost:27000"
registration_url = "https://localhost:55000"
```

Reply:
Fixed!